src/Stratis.Bitcoin/Configuration/Logging/LogLevel.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace Stratis.Bitcoin.Configuration.Logging
```

Comment:
This code is commented out, should this be removed?

Reply:
Removed.